vuepress/docs/next/docs/create/component-gen-overview.md

```diff
@@ -1,42 +1,44 @@
+- Begin with running software: By automating the creation of microservices and micro frontends, the user is able to bypass the manual development required to build the default configuration of an Entando project.
+- Customize as desired: The generated code is ready to be developed on and modified to enable features and capabilities.
+- Avoid a learning curve: The Entando Blueprint uses standard tools, technologies, and patterns that can be consumed by developers who are not familiar with the underlying technology.
+
+### Standardize Technology 
+
+- Guarantee technology compliance: The Component Generator encodes business standards for networking, monitoring, build tools, and distribution into the Entando Blueprint to ensure that business requirements are satisfied.
+- Guarantee internal compliance: The Entando Blueprint provides parameters to ensure the code complies with standards and practices of your organization including formatting, testing tools, static analysis tools, and package structures.
+- Jumpstart 3rd party adoption: Teams can quickly recognize and adopt code from system integrators and other third parties because these patterns and standards are encoded into the default application.
 
 ## Generated Component Flow
 
-This section provides an overview of the component generation process. If you want to jump into the code and a tutorial start here: [Entando Component Generator Tutorial](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md)
+The following provides an overview of the component generation process. To view the code and learn how to use the Entando Blueprint, refer to the [Entando Component Generator Tutorial](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md).
 
 1. Create components
-   - The Component Generator will create a Spring Boot microservice and optionally will create React micro frontends for entities that you add to your project
-   - The microservice and generated micro frontends are runnable out of the box and preconfigured to connect to [Entando Identity Management](../consume/identity-management.md)
-   - The code is meant to be edited and enhanced by developers. JHipster provides interactive updates and merges if you need to re-run the generator or add new functionality via JHipster
-   - The microservice includes liquibase and integrated data mocking if you want to generate data in your API calls
+   - The Component Generator creates a Spring Boot microservice and, optionally, React micro frontends for entities that you add to your project.
+   - The microservice and generated micro frontends are out-of-the-box runnable and preconfigured to connect to the [Entando Identity Management System](../consume/identity-management-system.md).
+   - To integrate user edits and enhancements to the code, JHipster offers interactive updates and merges to re-run the modified generator. Alternatively, the user can add new functionality via JHipster.
```

Comment:
can drop user to make first sentence flow nicer, especially since it's repeated in next sentence. not a big deal though

but the generator doesn't get modified, it's just generating with the new updates you give it

Reply:
the first instance of user needs to stay to make who is performing the changes explicit. 

the whole point of this bullet is that you can modify the generator, then get all your changes workable to run your new version.

Reply:
@nshaw, is it the blueprint or generator that get customized when using ECG

Reply:
@jyunmitch neither, it's the generated code that gets updated.  The blueprint/generator are the starting points and if you re-run (say by adding an entity or modifying the JDL), it's only the project code that gets modified, not the blueprint/generator itself.

Reply:
@jyunmitch i think the blueprint and generator are the same thing, right? i had been thinking of the generator as bigger than the blueprint, like implementing the blueprint, but that may not be an accurate distinction. the use of the two terms is a bit confusing tho. wrt the generator itself, the point of the bullet is that you make it your own through modifications. i'll just take out "modified" to not imply that you're tweaking the generator itself.

Reply:
Ok, I think the confusion is partly because there are two generators, kind of, the ECR and the JHipster blueprint, and the ECR uses the blueprint?

Reply:
let's pester @nshaw about this... pretty sure last standup when we asked if they were one and the same i shook my head and you nodded and he said they were synonymous...